WordPressKit/CommentServiceRemoteREST.m

```diff
@@ -400,12 +402,25 @@ - (void)getLikesForCommentID:(NSNumber *)commentID
                                      withVersion:ServiceRemoteWordPressComRESTApiVersion_1_2];
     NSNumber *siteID = self.siteID;
 
+    // If no count provided, default to endpoint max.
+    if (count == 0) {
+        count = @90;
+    }
+    
+    NSMutableDictionary *parameters = [NSMutableDictionary dictionaryWithDictionary:@{ @"number": count }];
+    
+    if (before != nil) {
```

Comment:
I like the explicit equality check. :)  `if (before) {` would suffice for a `nil` check but with fewer of us using objc with any frequency I think this makes the code more clear.